CHANGELOG.md

```diff
@@ -34,6 +34,9 @@ This changelog documents all notable changes to VAST and is updated on every rel
 - The new `aggregate` transform plugin allows for flexibly grouping and aggregating events. We recommend using it alongside the [`compaction` plugin](https://docs.tenzir.com/vast/features/compaction), e.g., for rolling up events into a more space-efficient representation after a certain amount of time.
   [#2076](https://github.com/tenzir/vast/pull/2076)
 
+- Add a parameter to the partition creation listener interface that allows plugins to get a view of the whole database state.
+  [#2103](https://github.com/tenzir/vast/pull/2103)
```

Comment:
Can you create a separate PR that fixes these for the final release next week? I'd like to go ahead with the rc2 rather than delay it until tomorrow. Same applies to the other two comments.